ethash/hash_types.h

```diff
@@ -0,0 +1,50 @@
+/* ethash: C/C++ implementation of Ethash, the Ethereum Proof of Work algorithm.
+ * Copyright 2018-2019 Pawel Bylica.
+ * Licensed under the Apache License, Version 2.0.
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+#ifdef __cplusplus
```

Comment:
Don't think this extern C block is needed any more

Reply:
> Don't think this extern C block is needed any more

You're probably right since there are no functions nor variables with external linkage in there. On the other side, there's no harm in having extern "C", we might as well leave it as is. (I took the files verbatim from https://github.com/chfast/ethash)

Reply:
Ok